CppCoreGuidelines.md

```diff
@@ -2621,8 +2622,33 @@ For passthrough functions that pass in parameters (by ordinary reference or by p
 
 **Enforcement**: ???
 
+<a name="Rf-decltype-forward"></a>
+### F.54: Use decltype on auto&& parameters to std::forward them.
```

Comment:
please move html inchor inside the header (before title) as has been done on master branch for all other headers.